cmd/controller/main.go

```diff
@@ -57,12 +65,17 @@ func main() {
 		eventingtls.TrustBundleLabelSelector,
 		sinks.JobSinkJobsLabelSelector,
 		eventtransform.JsonataResourcesSelector,
+		"app.kubernetes.io/name",
 	)
 
 	ctx = eventingfilteredfactory.WithSelectors(ctx,
 		eventtransform.JsonataResourcesSelector,
 	)
 
+	for _, inf := range []injection.InformerInjector{challenge.WithInformer, v1certificate.WithInformer, certificaterequest.WithInformer, clusterissuer.WithInformer, issuer.WithInformer} {
+		injection.Default.RegisterInformer(inf)
+	}
```

Comment:
If cert-manager is not installed, I think this logs errors and gets the controller stack on starting? Did you try that?

Also I don't think we need the `clusterissuer`, `issuer`, `challenge`, or `certificaterequest`

Reply:
yeah - forgot to comment that out